daemon/daemon.go

```diff
@@ -120,7 +121,11 @@ func (d *Daemon) UpdateProxyRedirect(e *endpoint.Endpoint, l4 *policy.L4Filter)
 		return 0, fmt.Errorf("can't redirect, proxy disabled")
 	}
 
-	log.Debugf("Adding redirect %+v to endpoint %d", l4, e.ID)
+	log.WithFields(log.Fields{
+		logfields.EndpointID: e.ID,
+		logfields.L4PolicyID: e.ProxyID(l4),
+		"obj":                logfields.Repr(l4),
```

Comment:
Nit: have obj as its own field in `logfields` package. Given that this is such a small nitpick, please create a micro task to do this so we can get this initial patch merged, as this is a non-blocking issue.

Reply:
Done! I needed to run through for some other changes anyway :) It is now `logfields.Object` but if you have a preferred name we can use that.